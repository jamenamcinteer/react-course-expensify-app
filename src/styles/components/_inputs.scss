.text-input {
  border: 1px solid #cacccd;
  height: 50px;
  font-size: $font-size-large;
  font-weight: 300;
  padding: 1rem;
}

.select {
  @extend .text-input;
}

.textarea {
  @extend .text-input;
  height: 10rem;
}
