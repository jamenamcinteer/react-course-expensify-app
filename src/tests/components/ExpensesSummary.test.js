import React from "react";
import { shallow } from "enzyme";
import { ExpensesSummary } from "../../components/ExpensesSummary";
import expenses from "../fixtures/expenses";

test("should render Expense Summary with no expenses", () => {
  const wrapper = shallow(
    <ExpensesSummary expenseCount={0} expensesTotal={0} />
  );
  expect(wrapper).toMatchSnapshot();
});

test("should render Expense Summary with one expense", () => {
  const wrapper = shallow(
    <ExpensesSummary expenseCount={1} expensesTotal={235} />
  );
  expect(wrapper).toMatchSnapshot();
});

test("should render Expense Summary with multiple expenses", () => {
  const wrapper = shallow(
    <ExpensesSummary expenseCount={23} expensesTotal={12304723001} />
  );
  expect(wrapper).toMatchSnapshot();
});
