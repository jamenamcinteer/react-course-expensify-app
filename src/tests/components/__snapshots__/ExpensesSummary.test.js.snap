// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense Summary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    23
     expense
    expenses
     totalling 
    $123,047,230.01
  </h1>
</div>
`;

exports[`should render Expense Summary with no expenses 1`] = `
<div>
  <h1>
    Viewing 
    0
     expense
    expenses
     totalling 
    $0.00
  </h1>
</div>
`;

exports[`should render Expense Summary with one expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     expense
    expense
     totalling 
    $2.35
  </h1>
</div>
`;
